downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md

```diff
@@ -0,0 +1,398 @@
+public type Person record {
+	string name;
+	float? score = 0;
+};
+public function main(int efficiency, string character, *Person person) {
+	// Process data here
+}
+```
+
+```bash
+bal run file.bal -- --name riyafa  100 --score=99.9 Good
+```
+
+This example, which is the same as above includes `100`, which gets mapped to `efficiency`, and `Good`, which gets mapped to `character` as operands. 
+
+Both operand and option parameters can be of types `int`, `float`, `decimal`, `string`, array of any of these types, and union of any of these types with `nil`. 
+
+Additionally, option parameters can be of types `boolean`, `boolean[]`, or `boolean?`.
+
+**Operand Arrays**
+
+>**Note:** If there is an operand parameter of type O[], then it cannot be followed by parameters of type O[], O?, and O x = d. Here O stands for a type that is a subtype of one of string, float, or decimal. An array value is specified by repeatedly specifying the `option` parameter.
+
+If `scores` is an int array, then,
+
+```bash
+bal run file.bal -- --scores=10 --scores=20 --scores=30
+```
+
+This produces the following int array.
+
+```bash
+[10, 20, 30]
+```
+
+**Option Boolean Parameters**
+
+When there’s an option of `boolean`, `boolean[]`, or `boolean?` type, it does not take an `option` argument. The presence of the option is considered to be `true` and the absence of it is considered to be false. 
+In the following example, suppose `results` is a boolean array.
+
+```bash
+bal run file.bal -- --results --results --results
+```
+
+This produces the following boolean array.
+
+```bash
+[true, true, true]
+```
+
+#### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Alpha5](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha5%22+label%3AType%2FBug+label%3ATeam%2FjBallerina).
+
+### Standard Library Updates
+
+#### New Features
+
+##### `graphql` Package
+
+- Added named fragment support
+- Added enum support
+- Added map support
+- Added compiler plugin validation for GraphQL services
+
+##### `email` Package
+Added compiler plugin validation for Email listener services.
+
+##### `udp` Package
+Added compiler plugin validation for UDP services.
+
+##### `http` Package
+Added compiler plugin validations for HTTP services.
+
+##### `websocket` Package
+Added compiler plugin validation for WebSocket services.
+
+##### `websub` Package
+Added compiler plugin validation for WebSub subscriber services.
+
+##### `websubhub` Package
+Added compiler plugin validation for WebSubHub services.
+
+##### `kafka` Package
+Added compiler plugin validations for Kafka services.
+
+##### `rabbitmq` Package
+Added compiler plugin validations for RabbitMQ services.
+
+##### `nats` Package
+Added compiler plugin validations for NATS services.
+
+##### `stan` Package
+Added compiler plugin validations for STAN services.
+
+##### `tcp` Package
+Added compiler plugin validation for TCP services.
```

Comment:
```suggestion
Added compiler plugin validation for services in the following packages: `http`, `grpc`, `websocket`, `websub`, `websubhub`, `graphql`, `kafka`, `rabbitmq`, `nats`, `stan`, `email`, `tcp`, `udp` and `file`

##### `graphql` Package

- Added named fragment support
- Added enum support
- Added map support
```

Reply:
Shall we order these alphabetically?
Added compiler plugin validation for services in the following packages: `email`, `file`, `graphql`, `grpc`,  `http`, `kafka`, `nats`, `rabbitmq`, `stan`, `tcp`, `udp`, `websocket`, `websub`, and `websubhub`.